### DISCUSSION TITLE
Node has shown their attitude of Ukraine

### DISCUSSION THREAD

**Discussion by Decodetalkers** [denoland/deno#13819] [General]
https://nodejs.org/en/

What's our attitude? If we can support Ukraine on deno land?

**Comment by Nugine** [comment]
<https://github.com/vuejs/vue/pull/12483#issuecomment-1054966381>
Deno is an OSS project. Please DO NOT be involved in politics.

**Reply by lucacasonato** [reply]
I want to push back on this: **open source is inherently very political**. An attack on open and free society is also an attack on open and free software. I am just not sure how much a banner on our website actually helps the people of Ukraine today.

Personally, I am very saddened by the continued Russian provoked war against Ukraine and it's people. I have donated to relief funds and humanitarian aid for the people of Ukraine, and I know others in the core team have too. This needless bloodshed needs to stop now.

**Reply by Nugine** [reply]
I personally sympathize with **the civilians of Ukraine and Russia**. But there are also other people in the world suffering from wars, pandemic, famine and poverty. It is impossible to help all of them directly.

An open source project is not a charitable organization. We should express our support by donating funds instead of putting up a small banner. It's just nonsense.

You can publish a blog saying that the members of Deno core team have donated xxx dollars to help the civilians affected by the war. But, please, do not bring political messages to the project itself, unless all of the contributors agree with that.

**Reply by David-Else** [reply]
@lucacasonato I would say FOSS is political, open source is the 'de-politicized' business friendly profit driven version. Not that there is anything wrong with that.

**Comment by Nugine** [comment]
nodejs has reverted the banner commit
https://github.com/nodejs/nodejs.org/pull/4467
